Weather/Core/Entities/TimedForecast.swift

```diff
@@ -18,9 +18,9 @@ class TimedForecast {
     }
     
     func getTime() -> String {
```

Comment:
No need for such methods. This is not Java. You can remove private from properties and use them as they are.

Reply:
@pkondrashkov But it's violation of OOP concept: encapsulation, isn't it?

Reply:
Nope, this is not an encapsulation. Encapsulation is about hiding internal implementation behind private methods \ variables. This is just a property, you can think of it like a setter\getter with a variable attached to it already.

It is completely okay and swifty way to write the code.